baseapp/abci.go

```diff
@@ -317,6 +323,51 @@ func (app *BaseApp) Query(req abci.RequestQuery) abci.ResponseQuery {
 	return sdkerrors.QueryResult(sdkerrors.Wrap(sdkerrors.ErrUnknownRequest, "unknown query path"))
 }
 
+func handleQueryGRPC(app *BaseApp, handler GRPCQueryHandler, req abci.RequestQuery) abci.ResponseQuery {
```

Comment:
You pass in `*BaseApp` as an arg, when it should rather be defined as a method on `*BaseApp`.
```suggestion
func (app *BaseApp) handleQueryGRPC(handler GRPCQueryHandler, req abci.RequestQuery) abci.ResponseQuery {
```